cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-cli-guide.adoc

```diff
@@ -0,0 +1,426 @@
+I: To determine when your cluster is Ready, run 'rosa describe cluster -c my-rosa-cluster'.
+I: To watch your cluster installation logs, run 'rosa logs install -c my-rosa-cluster --watch'.
+----
++
+[NOTE]
+====
+The cluster state will remain as “waiting” until the next two steps are completed.
+====
+
+=== Creating Operator roles
+
+. At the end of the output in the above step, we are told the next commands to run. These roles need to be created _once_ for _each_ cluster. To create the roles run the following command:
++
+[source,terminal]
+----
+rosa create operator-roles --mode manual --cluster <cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Run the following commands to create the operator roles:
+    aws iam create-role \
+        --role-name my-rosa-cluster-openshift-image-registry-installer-cloud-credentials \
+        --assume-role-policy-document file://operator_image_registry_installer_cloud_credentials_policy.json \
+        --tags Key=rosa_cluster_id,Value=1mkesci269png3tck000000000000000 Key=rosa_openshift_version,Value=4.8 Key=rosa_role_prefix,Value= Key=operator_namespace,Value=openshift-image-registry Key=operator_name,Value=installer-cloud-credentials
+
+    aws iam attach-role-policy \
+        --role-name my-rosa-cluster-openshift-image-registry-installer-cloud-credentials \
+        --policy-arn arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-image-registry-installer-cloud-creden
+    [...]
+----
+
+. Run each of the `aws` commands.
+
+=== Creating the OIDC provider
+. Run the following to create the OIDC provider:
++
+[source,terminal]
+----
+rosa create oidc-provider --mode manual --cluster <cluster-name>
+----
+
+. This displays the `aws` commands that you need to run. Run the commands like the below example:
++
+[source,terminal]
+----
+I: Run the following commands to create the OIDC provider:
+$ aws iam create-open-id-connect-provider \
+--url https://rh-oidc.s3.us-east-1.amazonaws.com/1mkesci269png3tckknhh0rfs2da5fj9 \
+--client-id-list openshift sts.amazonaws.com \
+--thumbprint-list a9d53002e97e00e043244f3d170d000000000000
+
+$ aws iam create-open-id-connect-provider \
+--url https://rh-oidc.s3.us-east-1.amazonaws.com/1mkesci269png3tckknhh0rfs2da5fj9 \
+--client-id-list openshift sts.amazonaws.com \
+--thumbprint-list a9d53002e97e00e043244f3d170d000000000000
+----
+
+. Your cluster will now continue the installation process.  
+
+=== Checking the installation status
+. Run one of the following commands to check the status of your cluster:
++
+* For a detailed view of the status, run:
++
+[source,terminal]
+----
+rosa describe cluster --cluster <cluster-name>
+----
+* For an abridged view of the status, run:
++
+[source,terminal]
+----
+rosa list clusters
+----
+
+The cluster state will change from “waiting” to “installing” to "ready". This will take about 40 minutes.
+
+. Once the state changes to “ready” your cluster is installed.
```

Comment:
Not sure this should be numbered? Made similar comment earlier in the review....